sdwan/resource_sdwan_ntp_feature_template.go

```diff
@@ -51,24 +51,132 @@ func resourceSDWANNtpFeatureTemplate() *schema.Resource {
+						"vedge-C8200-1N-4T",
+						"vedge-ISR-4331",
+						"vedge-C1127X-8PMLTEP",
+						"vedge-C1117-4PMLTEEAWE",
+						"vedge-ISR-4451-X",
+						"vedge-C1113-8PLTEEA",
+						"vedge-IR-1821",
+						"vedge-ASR-1001-X",
+						"vedge-ISR-4321",
+						"vedge-C1116-4PLTEEAWE",
+						"vedge-C1109-4PLTE2P",
+						"vedge-C1121-8P",
+						"vedge-ASR-1002-HX",
+						"vedge-C1111-8PLTEEAW",
+						"vedge-C1112-8PWE",
+						"vedge-C1101-4PLTEP",
+						"vedge-ISR1100-4GLTENA-XE",
+						"vedge-C1111-8PLTELA",
+						"vedge-IR-1835",
+						"vedge-C1121X-8PLTEP",
+						"vedge-IR-1833",
+						"vedge-C8300-1N1S-4T2X",
+						"vedge-C1121-4P",
+						"vedge-ISR-4351",
+						"vedge-C1117-4PLTELA",
+						"vedge-C1116-4PWE",
+						"vedge-nfvis-C8200-UCPE",
+						"vedge-C1113-8PM",
+						"vedge-IR-1831",
+						"vedge-C1127-8PLTEP",
+						"vedge-C1121-8PLTEPW",
+						"vedge-C1113-8PW",
+						"vedge-ASR-1001-HX",
+						"vedge-C1128-8PLTEP",
+						"vedge-C1113-8PLTEEAW",
+						"vedge-C1117-4PW",
+						"vedge-C1116-4P",
+						"vedge-C1113-8PMLTEEA",
+						"vedge-C1112-8P",
+						"vedge-ISR-4461",
+						"vedge-C1116-4PLTEEA",
+						"vedge-ISR-4221",
+						"vedge-C1117-4PM",
+						"vedge-C1113-8PLTELAWZ",
+						"vedge-C1117-4PMWE",
+						"vedge-C1109-2PLTEVZ",
+						"vedge-C1113-8P",
+						"vedge-C1117-4P",
+						"vedge-nfvis-ENCS5400",
+						"vedge-C8300-2N2S-6T",
+						"vedge-C1127-8PMLTEP",
+						"vedge-ESR-6300",
+						"vedge-ISR-4221X",
+						"vedge-ISR1100-4GLTEGB-XE",
+						"vedge-C8500-12X",
+						"vedge-C1109-2PLTEGB",
+						"vedge-CSR-1000v",
+						"vedge-C1113-8PLTEW",
+						"vedge-C1121X-8PLTEPW",
+						"vedge-ISR1100-6G-XE",
+						"vedge-C1121-4PLTEP",
+						"vedge-C1111-8PLTEEA",
+						"vedge-C1117-4PLTEEA",
+						"vedge-C1127X-8PLTEP",
+						"vedge-C1109-2PLTEUS",
+						"vedge-C1112-8PLTEEAWE",
+						"vedge-C1161X-8P",
+						"vedge-C8500L-8S4X",
+						"vedge-C1111-8PW",
+						"vedge-C1161X-8PLTEP",
+						"vedge-C1101-4PLTEPW",
+						"vedge-ISR1100X-4G-XE",
+						"vedge-IR-1101",
+						"vedge-C1111-4P",
+						"vedge-C1111-4PW",
+						"vedge-C1111-8P",
+						"vedge-C1117-4PMLTEEA",
+						"vedge-C1113-8PLTELA",
+						"vedge-C1111-8PLTELAW",
+						"vedge-C1161-8PLTEP",
+						"vedge-ISR1100X-6G-XE",
+						"vedge-ISR-4431",
+						"vedge-C1101-4P",
+						"vedge-C1109-4PLTE2PW",
+						"vedge-C1113-8PMWE",
+						"vedge-C1118-8P",
+						"vedge-C1126-8PLTEP",
+						"vedge-C8300-1N1S-6T",
+						"vedge-C1121-8PLTEP",
+						"vedge-C8300-2N2S-4T2X",
+						"vedge-C1112-8PLTEEA",
+						"vedge-C1111-4PLTELA",
+						"vedge-ASR-1002-X",
+						"vedge-C1111X-8P",
+						"vedge-C1126X-8PLTEP",
+						"vedge-ASR-1006-X",
+						"vedge-C8000V",
+						"vedge-ISR1100-4G-XE",
+						"vedge-C1117-4PLTELAWZ",
+						"vedge-ISRv",
 					}, false),
 				},
 			},
+			"template_type": {
+				Type:     schema.TypeString,
+				Required: true,
+				ValidateFunc: validation.StringInSlice(
```

Comment:
We should make this termplate_type as ForceNew "true" as we can not update the template type after creation. Please update that and test that as well

Reply:
added ForceNew "true" attribute in termplate_type variable